packages/terra-clinical-onset-picker/src/OnsetPicker.jsx

```diff
@@ -0,0 +1,427 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import 'terra-base/lib/baseStyles';
+import Field from 'terra-form/lib/Field';
+import Fieldset from 'terra-form/lib/Fieldset';
+import NumberField from 'terra-form/lib/NumberField';
+import DatePicker from 'terra-date-picker/lib/DatePicker';
+import SelectField from 'terra-form/lib/SelectField';
+
+
+const propTypes = {
+  /**
+   * The ISO 8601 string representation of the birth date to calculate an onset date from.
+   */
+  birthdate: PropTypes.string.isRequired,
+
+  /**
+   * The granularity of the onset date. YEAR, MONTH, and DATE are accepted.
+   */
+  granularity: PropTypes.string,
+
+  /**
+   * Name of the granularity select. The name should be unique.
+   */
+  granularitySelectName: PropTypes.string.isRequired,
+
+  /**
+   * A callback function to execute when a granularity is selected.
+   * The first parameter is the event. The second parameter is the changed granularity value.
+   */
+  granularitySelectOnChange: PropTypes.func,
+
+  /**
+   * The precision of the onset date. ABOUT, BEFORE, AFTER, and UNKNOWN are accepted.
+   */
+  precision: PropTypes.string,
```

Comment:
Update the prop type as I mentioned in the tech design: https://github.com/cerner/terra-clinical/issues/186

Reply:
Updated per comments on the tech design.